TombEngine/Math/Random.cpp

```diff
@@ -26,54 +26,112 @@ namespace TEN::Math::Random
 
 	Vector2 GenerateDirection2D()
 	{
-		auto vector = Vector2(GenerateFloat(-1.0f, 1.0f), GenerateFloat(-1.0f, 1.0f));
-		vector.Normalize();
-		return vector;
+		float angle = GenerateFloat(0.0f, PI_MUL_2); // Generate angle in full circle.
```

Comment:
Wouldn't this approach be slower than just generating from `(-1, 1)` range directly? Although if this function will be rarely used, it shouldn't be a big problem.

Reply:
Using the old `(-1, 1)` range would result in a bias toward directions closer to square corners, which is what I wanted to avoid in the 3D variant of this function.